modules/misc/gtk.nix

```diff
@@ -166,14 +166,30 @@ in {
     home.packages = optionalPackage cfg.font ++ optionalPackage cfg.theme
       ++ optionalPackage cfg.iconTheme;
 
-    home.file.${cfg2.configLocation}.text =
-      concatStringsSep "\n" (mapAttrsToList formatGtk2Option ini) + "\n"
-      + cfg2.extraConfig;
+    home.file.${cfg2.configLocation} = {
+      text = concatStringsSep "\n" (mapAttrsToList formatGtk2Option ini) + "\n"
+        + cfg2.extraConfig;
+      onChange = let xsettingsd = "${pkgs.xsettingsd}/bin/xsettingsd";
+      in ''
+        DISPLAY=":0" $DRY_RUN_CMD ${xsettingsd} -c \
```

Comment:
Pinning DISPLAY is not a possibility.

DISPLAY should normally be available in the HM running environment in the onChange.
It might not be available when running the activation script from the NixOS module (because HM is then setup as a systemd service and lacks most user environment variables).

Reply:
Can xsettingsd be launched even if another instance is already running?

Reply:
Oops, leftover hardcoding from local tests haha.

I wonder if there's any way to get get the variable in the NixOS module too?

Reply:
Kinda hacky, not sure if it's robust enough. But i was able to grep it out of `/run/user/$UID/env-vars`.